### ISSUE TITLE
Release binary for ARM (Apple Silicon)

### ISSUE THREAD

**Issue by thii** [SwiftGen/SwiftGen#805]
It'd be nice if an arm64 or a universal binary is provided for the latest release.

**Comment by djbe** [comment]
Depends on your installation method:
- Cocoapods: not universal yet, will have to look into it.
- SPM/Mint: should already be supported, as it's compiled on your device
- Homebrew: impossible to do until Homebrew itself has complete arm64 support.

**Comment by cabeca** [comment]
Homebrew has been updated to 3.0.0 with support for Apple Silicon.

**Comment by vani2** [comment]
For me that's not working.
```
brew update
brew install swiftgen

Error: swiftgen: no bottle available!
You can try to install from source with:
  brew install --build-from-source swiftgen
Please note building from source is unsupported. You will encounter build
failures with some formulae. If you experience any issues please create pull
requests instead of asking for help on Homebrew's GitHub, Twitter or any other
official channels.
```

**Comment by FelixLisczyk** [comment]
The Homebrew error is also mentioned in #811 and #820. Maybe these issues should be merged?

**Comment by ricardopereira** [comment]
I confirm it works with Mint:

```
➜  ~ mint install SwiftGen/SwiftGen
🌱 Finding latest version of SwiftGen
🌱 Cloning SwiftGen 6.4.0
🌱 Resolving package
🌱 Building package
🌱 Copying resources for SwiftGen: templates, Resources/Stencil-Info.plist, Resources/StencilSwiftKit-Info.plist, Resources/SwiftGen-Info.plist, Resources/SwiftGenKit-Info.plist ...
🌱 Installed SwiftGen 6.4.0
🌱 Linked swiftgen 6.4.0 to /Users/<redacted>/.mint/bin
```

**Comment by vani2** [comment]
> I confirm it works with Mint:

I've got error 

```
~ % mint install SwiftGen/SwiftGen
🌱 Finding latest version of SwiftGen
🌱 Cloning SwiftGen 6.4.0
🌱 Resolving package
xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun
🌱 Encountered error during "swift package resolve". Use --verbose to see full output
🌱  Failed to resolve SwiftGen 6.4.0 with SPM
```

**Comment by ricardopereira** [comment]
@vani2 Do you have Xcode installed?

**Comment by vani2** [comment]
Yes, I had, but there was no selected CLT in settings. And it has built my project w/o any error though.  After selecting, SwiftGen was installed OK. Thanks.

**Comment by Jonatthu** [comment]
@vani2 Where I can find this CLT settings?

**Comment by vani2** [comment]
> @vani2 Where I can find this CLT settings?

<img width="942" alt="Screenshot 2021-04-07 at 13 02 57" src="https://user-images.githubusercontent.com/4734983/113848998-b5c44600-97a1-11eb-9feb-c6f646f9e7de.png">

**Comment by Jonatthu** [comment]
@vani2 thanks a lot!

**Comment by oleiade** [comment]
I'm running into the same issue:
```
❯ brew install swiftgen
Error: swiftgen: no bottle available!
You can try to install from source with:
  brew install --build-from-source swiftgen
Please note building from source is unsupported. You will encounter build
failures with some formulae. If you experience any issues please create pull
requests instead of asking for help on Homebrew's GitHub, Twitter or any other
official channels.
```

For your information, I did `brew update` (3.1.7) and `brew upgrade` before trying to install.

**Comment by mudithsilva** [comment]
I checked in Mint installation and it's working fine on Apple Silicon.
Try:-

To install Mint:- brew install mint
To Install SwiftGen:- sudo mint install SwiftGen/SwiftGen
To manually run the SwiftLint and SwiftGen (navigate to project folder) :- mint run swiftgen

**Comment by oleiade** [comment]
@mudithsilva That's all fine and dandy, but I do not want to use Mint 👍🏻

**Comment by Linquas** [comment]
Any update on this topic?

**Comment by pawisoon** [comment]
Any update on this? It would be great to have this binary released :)

**Comment by andrewsexcellent** [comment]
Any update on this

**Comment by foxsin10** [comment]
Any update on this?
Get compile error with Xcode 13

**Comment by EvanCooper9** [comment]
I've been able to get around this temporarily by building from source `brew install swiftgen --build-from-source`.

I use a Brewfile for my project, so: `brew "swiftgen", args: ["build-from-source"]`

**Comment by djbe** [comment]
Implemented in SwiftGen 6.5.0, which has been released 🎉

~For Homebrew, the PR to update the formula is here: https://github.com/Homebrew/homebrew-core/pull/86408. Hopefully it'll be merged soon.~ Update: also released on Homebrew.
